ibm/config_functions.go

```diff
@@ -46,10 +46,6 @@ func getBaseURL(region string) string {
  *
  */
 func setupOpenWhiskClientConfig(namespace string, c *bluemix.Config, wskClient *whisk.Client) (*whisk.Client, error) {
-	if c.UAAAccessToken == "" && c.UAARefreshToken == "" {
-		return nil, fmt.Errorf("Couldn't retrieve auth key for IBM Cloud Function")
```

Comment:
We already merged this PR why again it showing in diff

Reply:
Rebase your branch with latest of master

Reply:
Rebased master branch and fixed the issue.